### PR TITLE
Edits to automation chapter

### PR DESCRIPTION
**Pull request by gvwilson**
Minor fixes to wording - no significant content changes.

Requires #394.

### CODE REVIEW

File: py-rse/automate.Rmd [hunk 1]
```diff
@@ -79,36 +72,30 @@ Make works as follows:
...
-> and [drake][drake] (for R).
-> If you want to go deeper,
-> [@Smit2011] describes the design and implementation of several build tools.
-
-### Setting Up
-
 This chapter uses a version of Make called [GNU Make][gnu-make].
 It comes with macOS and Linux,
-and you can install it on Windows using [Chocolatey][chocolatey]:
+and you can install it on Windows using [Chocolatey][chocolatey] by running:
```
Comment (DamienIrving) [inline py-rse/automate.Rmd:77]: Is this bit on installing make consistent with / should it refer back to the appendix where we will have all the installation instructions?

File: py-rse/automate.Rmd [hunk 2]
```diff
@@ -79,36 +72,30 @@ Make works as follows:
...
 
+> **Keep Tracking With Version Control**
+>
+> We learned about tracking our project's version history using Git
+> in Chapters \@ref(git-cmdline) and \@ref(git-advanced).
+> We encourage you to continue apply the Git workflow
+> throughout the rest of our code development,
+> though we won't continue to remind you.
+
 ### Acknowledgements
```
Comment (DamienIrving) [inline py-rse/automate.Rmd:91]: We initially decided to put acknowledgements at the end of each chapter #332, but I think after that we decided to put them all in one place somewhere?

File: py-rse/automate.Rmd (hunk 1 above)
Comment (gvwilson) [reply py-rse/automate.Rmd:77]: Moved to `install.Rmd`.

File: py-rse/automate.Rmd (hunk 2 above)
Comment (gvwilson) [reply py-rse/automate.Rmd:91]: Consolidated in introduction.

**Pull request merged**